.github/workflows/main.yml

```diff
@@ -97,18 +97,16 @@ jobs:
           misc/loc
 
       - name: install Bats
-        # We need (1) old Bats, not bats-core and (2) prior to commit 1735a4f,
-        # because this is what is provided in distros we need to support and
-        # it contains a bug we work around (see issue #552).
+        # Previously, we installed old Bats here. Changing this to bats-core
+        # (see issue #582).
         run: |
           cd /usr/local/src
-          git clone --depth 1 --branch v0.4.0 https://github.com/sstephenson/bats.git
-          cd bats
+          git clone --depth 1 --branch v1.2.1 https://github.com/bats-core/bats-core.git
```

Comment:
What's the reasoning to install from Git rather than via `apt`? In fact you might be able to remove this step entirely and install Bats above as a generic dependency.